spec/requests/dois_spec.rb

```diff
@@ -3081,9 +3087,19 @@
       end
 
       it 'updates the Doi' do
-        patch "/dois/#{doi.doi}", update_attributes, headers
+        patch "/dois/#{doi.doi}?license=CC0-1.0", update_attributes, headers
```

Comment:
I'm not sure if I don't understand this, Could you help me to understand why the licence need to passed as an attribute in this test ?

Reply:
This is testing that the query parameter for license works.

Reply:
Thanks for catching this. Will fix.